### DISCUSSION TITLE
Win GUI: Different Colors for Download/Seed Progress Bars

### DISCUSSION THREAD

**Discussion by pliny** [transmission/transmission#5351] [Ideas]
On the Windows GUI, the progress bars for downloading and seeding are the same shade of green. It is not immediately clear if the download has finished when glancing at the progress bar (with notifications off). It would be nice if the progress bar would change color when it started seeding so finished downloads would be immediately recognizable.

**Comment by macoloco77** [comment]
i am lookinf for a feature like this long time....i was stuck on 2.8.4 last version that shows different colors for this

blue = downloading
green= completed and seeding
red = warning or error

**Comment by Pentaphon** [comment]
> On the Windows GUI, the progress bars for downloading and seeding are the same shade of green. It is not immediately clear if the download has finished when glancing at the progress bar (with notifications off). It would be nice if the progress bar would change color when it started seeding so finished downloads would be immediately recognizable.

You make a good point I should have seen long ago. How about we just make it the same as the Mac client?

Edit: made an issue here: https://github.com/transmission/transmission/issues/5648

![screenshot_apple_light](https://github.com/transmission/transmission/assets/49374058/f9d7dd5a-7920-4641-a9d2-6988234784b8)
